Dockerfile

```diff
@@ -4,8 +4,11 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 ADD dependencies.apt.txt ./
 
+RUN printf "deb-src http://deb.debian.org/debian bullseye main\ndeb-src http://deb.debian.org/debian-security/ bullseye-security main\ndeb-src http://deb.debian.org/debian bullseye-updates main" >> /etc/apt/sources.list
+
 RUN apt-get update && \
     xargs apt-get install --no-install-recommends -y < dependencies.apt.txt && \
+    apt-get --no-install-recommends -y build-dep python3-apt && \
```

Comment:
It feels a bit weird to install the build-deps from apt, but then install the package via pip. I guess that's fine because it makes the `requirements.txt` usable on non-Debian distros.

Reply:
I noticed it too, that pip modules are getting installed twice one by apt then by pip.

Not sure what to do there.